framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -95,6 +95,9 @@ Licensed to the Apache Software Foundation (ASF) under one
     protected ArrayList<PluginEntry> pluginEntries;
     protected CordovaInterfaceImpl cordovaInterface;
 
+    //flag for case when appView is null during onStart
+    private boolean shouldHandleStartOnAppViewInit = false;
```

Comment:
My tests pass without it.  I'll remove it.  I'll also remove the similar flag from CordovaWebViewImpl.